mod/03_machines/lib/javascript/wagn.js.coffee

```diff
@@ -150,6 +150,9 @@ $(window).ready ->
   $('body').on 'ajax:success', '.slotter', (event, data, c, d) ->
     unless event.slotSuccessful
       $(this).slotSuccess data
+      # should scroll to top after clicking on new page
+      if $(this).hasClass "card-paging-link"
+        $("body").scrollTop 0
```

Comment:
wait, they don't necessarily want to scroll to the top of the page, they want to scroll to the top of the list (the slot), right?  Remember that the thing we're paging through might be waaaay down the page.